eras/alonzo/impl/src/Cardano/Ledger/Alonzo/TxWits.hs

```diff
@@ -633,3 +711,6 @@ alonzoEqTxWitsRaw txWits1 txWits2 =
   shelleyEqTxWitsRaw txWits1 txWits2
     && eqRawType (txWits1 ^. datsTxWitsL) (txWits2 ^. datsTxWitsL)
     && eqRawType (txWits1 ^. rdmrsTxWitsL) (txWits2 ^. rdmrsTxWitsL)
+
+mapListDecoderA :: Decoder s [Annotator a] -> ([a] -> m b) -> Decoder s (Annotator (m b))
+mapListDecoderA decList transformList = fmap transformList . sequence <$> decList
```

Comment:
My suggestion about `NonEmpty` will require this slight change (also `m` was unnecessary):
```suggestion
mapListDecoderA :: Foldable f => Decoder s (f (Annotator a)) -> (f a -> b) -> Decoder s (Annotator b)
mapListDecoderA decList transformList = fmap transformList . sequence . toList <$> decList
```